Hospital.Application/Services/AnalyticsService.cs

```diff
@@ -0,0 +1,69 @@
+﻿using AutoMapper;
+using Hospital.Application.Contracts;
+using Hospital.Application.Contracts.Appointments;
+using Hospital.Application.Contracts.Doctors;
+using Hospital.Application.Contracts.Patients;
+using Hospital.Domain.Interfaces;
+
+namespace Hospital.Application.Services;
+
+/// <summary>
+/// Provides analytics over doctors, patients, and appointments using repository abstractions.
+/// </summary>
+public class AnalyticsService(
+    IDoctorRepository doctors,
+    IAppointmentRepository appointments,
+    IMapper mapper) : IAnalyticsService
+{
+    public IEnumerable<DoctorDto> GetDoctorsWith10Plus() =>
+        doctors.GetAll()
+               .Where(d => d.ExperienceYears >= 10)
+               .Select(mapper.Map<DoctorDto>);
```

Comment:
Выровнять код тут и ниже.

```suggestion
        doctors.GetAll()
             .Where(d => d.ExperienceYears >= 10)
             .Select(mapper.Map<DoctorDto>);
```